homeassistant/components/adguard/__init__.py

```diff
@@ -197,6 +197,11 @@ class AdGuardHomeDeviceEntity(AdGuardHomeEntity):
     @property
     def device_info(self) -> DeviceInfo:
         """Return device information about this AdGuard Home instance."""
+        if self.adguard.tls:
+            config_url = f"https://{self.adguard.host}/"
+        else:
+            config_url = f"http://{self.adguard.host}/"
```

Comment:
This will not work for add-ons. Check the source of the configuration entry and see if we need to redirect it to the add-on ingress interface.

Reply:
Should be fixed now, thanks frenck!